test/performance/src/artilleryfunc.js

```diff
@@ -1,11 +1,70 @@
 const config = require('./config')
+const axios = require('axios')
 
 module.exports = {
 	setLoginBody: setLoginBody,
 	logAfterResponse: logAfterResponse,
 	authLogin: authLogin,
 	setAuthHeader: setAuthHeader,
 	sessionDataKey: sessionDataKey,
+	loginToWso2: loginToWso2,
+	logBeforeRequest: logBeforeRequest,
+	useCookie: useCookie,
+}
+
+let testCookie = {}
+let triedLogin = false
+
+function useCookie(requestParams, context, ee, next) {
```

Comment:
Is this ever used? There's a bug - `requestParams.headers.cookie = {'lpg-ui': testCookie}` should be `requestParams.headers.cookie = testCookie`. But if the function isn't used just remove.